python/grass/pygrass/modules/interface/module.py

```diff
@@ -555,7 +555,7 @@ def __init__(self, cmd, *args, **kargs):
         except OSError as e:
             print("OSError error({0}): {1}".format(e.errno, e.strerror))
             str_err = "Error running: `%s --interface-description`."
-            raise GrassError(str_err % self.name)
+            raise GrassError(str_err % self.name) from e
```

Comment:
```suggestion
            raise GrassError(str_err % self.name) from None
```
Maybe here, and in similar cases too?

Reply:
The code around here is weird, we shouldn't be using print for this there. (It was out of scope of the PR, so I wasn't changing everything, in order to be reviewable). But None makes sense, as OSError is only to know what that we couldn't find the module (or fails), it's not the real exception.

Reply:
You can get PermissionError or FileNotFoundError which is likely a relevant info. The `from e` will preserve that, no? ...if yes, that would be an improvement. Semantic of `GrassError` is not clear to me and `Error running:` is more hiding than explaining what happened here. The `print` seems to assume some usage and tries to add convenience. ...yep, some deeper revision would be nice.